### PR TITLE
Hotfix release version 0.10.1

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

File: releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Headers/zcashlc.h [hunk 1]
```diff
@@ -1022,7 +1022,7 @@ bool zcashlc_seed_fingerprint(const uint8_t *seed,
  *
  * # Safety
  *
- * - `safe_rewind_ret` must be non-null, aligned, and valid for writing a `uint32_t`.
+ * - `safe_rewind_ret` must be non-null, aligned, and valid for writing an `int64_t`.
```
Comment (str4d) [inline releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Headers/zcashlc.h:1025]: Why is this comment changing when the corresponding comment in the Rust code was unaltered by this PR?

**Review by str4d — APPROVED** [review]
utACK 7f84363b67593615ce51d1a2888f358870b42835 with question. I did not check the build artifacts.

File: releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Headers/zcashlc.h (hunk 1 above)
Comment (nuttycom) [reply releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Headers/zcashlc.h:1025]: The fix to the comment was merged after the last release was built.

**Review by str4d — APPROVED** [review]
Re-utACK ea902e98471587942b8508569384f8dbe69b7553

**Pull request merged**